src/matlab2tikz.m

```diff
@@ -1502,6 +1502,16 @@ case guitypes()
 
     options = setAxisTicks(m2t, options, axis, pgfTicks, pgfTickLabels, ...
         hasMinorTicks, tickDirection, isDatetimeTicks);
+        
+    options = setAxisTickLabelStyle(m2t, options,axis,handle);
+end
+% ==============================================================================
+function options = setAxisTickLabelStyle(m2t, options, axis, handle)
+    keywordTickLabelRotation = [upper(axis), 'TickLabelRotation'];
+    Rotation = getOrDefault(handle, keywordTickLabelRotation, 0);
```

Comment:
@miscco I disagree: introducing a variable (even if it's used only once) is a good thing because it makes the code self-documenting.
I'm also more in favor of `rotation` rather than `tickLabelRotation`: it is more terse and the scope of the function implies we are considering tick labels and not the rotation of something else (in a more generic function, however, I would totally agree with you to specify which rotation).